styles/globals.css

```diff
@@ -54,6 +54,22 @@ a {
   text-decoration: none;
 }
 
+h2 {
+  margin: 0;
+  font-size: 2.7em;
+  font-weight: bold;
+  padding: 0.05em 0.2em;
+  line-height: 1em;
+  background-image: linear-gradient(
+      var(--secondary-color),
+      var(--secondary-color)
+    ),
+    linear-gradient(var(--secondary-color), var(--secondary-color));
+  background-size: 3px 58%, 52% 3px;
```

Comment:
Are you sure you want to use px instead of something responsive?

Reply:
In this case - yes :)
It's the border element, which should not scale.